src/components/Nav.tsx

```diff
@@ -4,6 +4,7 @@ import { Home, Menu, X } from "react-feather";
 import tw from "twin.macro";
 import { Link } from "./Link";
 import { Logo } from "./Logo";
+import { OpenSearchModalButton } from "@/components/Search";
```

Comment:
Super duper tiny nit: but we may wanna standardize the way we do imports on this repo moving forward. Maybe best as an incoming todo.

Reply:
Yeah. I've defaulted to `@/...` in this PR, but didn't want to touch anything irrelavent to reduce churn.

Reply:
#244